frontend/src/components/common/info/messages/EmptyUsersMessage.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Center, useStyleConfig } from "@chakra-ui/react";
+
+import DisplayStudentsIllustration from "../../../../assets/illustrations/display-students.svg";
+import MessageContainer from "../MessageContainer";
+
+const EmptyUsersMessage = ({ role }: { role: string }): React.ReactElement => {
+  const styles = useStyleConfig("Center", { variant: "emptyMessage" });
+  return (
+    <Center __css={styles}>
+      <MessageContainer
+        image={DisplayStudentsIllustration}
```

Comment:
not the most fitting image, but i don't think we have another design for this

Reply:
Fair! Let's add a backlog ticket to ask design for a better image, but I'm fine with keeping this image for the mvp